src/compute/src/render/reduce.rs

```diff
@@ -702,12 +734,43 @@ where
                             );
                         }
                     }
+                    // After complaining, output an error here so that we can eventually
+                    // report it in an error stream.
+                    target.push((Err(format!("Invalid data in source, saw negative accumulation for key {key:?} in hierarchical mins-maxes aggregate")), 1));
```

Comment:
Nit: This line is a bit too long for my taste. I guess rustfmt has given up at this point :D

Reply:
I think you are right that it gave up. :-D  After I broke the line here and in the `warn!` messages above and in `build_bucketed`, some reformatting actually happened.